apps/server/src/shared/repo/task/task.repo.integration.spec.ts

```diff
@@ -196,6 +227,46 @@ describe('TaskRepo', () => {
 				expect(total).toEqual(0);
 				expect(result).toHaveLength(0);
 			});
+
+			it('should find private tasks created by the user', async () => {
```

Comment:
In this test file we have the sorting the test by the possible parameters and filters. 
We can change the sorting, but then we should do it for the complet file, otherwise we have a mix and it is hard to figure out what is tested.